core/trino-main/src/main/java/io/trino/dispatcher/DispatchExecutor.java

```diff
@@ -18,7 +18,7 @@
 import com.google.common.util.concurrent.ListeningScheduledExecutorService;
 import io.airlift.concurrent.ThreadPoolExecutorMBean;
 import io.trino.execution.QueryManagerConfig;
-import io.trino.version.EmbedVersion;
+import io.trino.spi.VersionEmbedder;
```

Comment:
Why SPI?

Reply:
This is where the interface is declared.

Reply:
Oh, I though it was a new thing in this PR. Nevermind :)